src/StartupScripts/OsServiceScriptManager.ts

```diff
@@ -0,0 +1,44 @@
+import { StartupManager } from './StartupManager';
+import { StartupOptions } from './StartupOptions';
+import service from 'os-service';
+import { promisify } from 'util';
+
+export class OsServiceScriptManager implements StartupManager {
+  public install(opts: StartupOptions): Promise<void> {
+    return new Promise<void>((resolve, reject) => {
+      service.add(
+        opts.serviceName,
+        {
+          programArgs: opts.exeArgs,
+          programPath: opts.exePath
+        },
+        (error: Error) => {
+          if (error) {
+            return reject(error);
+          }
+          resolve();
+        }
+      );
+    });
```

Comment:
Why don't use `promisify` here? Especially when it is already used below

```suggestion
    return promisify(service.add)(
      opts.serviceName,
      {
        programArgs: opts.exeArgs,
        programPath: opts.exePath
      }
    );
```